platform/mellanox/hw-management-wd.sh

```diff
@@ -46,7 +46,7 @@ CPLD_LPC_WD2_ACT_REG=0xcf
 CPLD_LPC_WD_CPBLT_BIT=6
 CPLD_LPC_WD_RESET_CAUSE_BIT=6
 CPLD_LPC_WD_RESET=1
-WD3_DFLT_TO=600
+WD3_DFLT_TO=180
```

Comment:
This script is a backport from newer hw-management package. Chaning it here means we need to change default in hw-management package in master. I suggest to change this in sonic-utlitiies, you can call this script like this:
```
hw-management-wd.sh start 180
```

Reply:
got it.